exopy_hqc_legacy/instruments/drivers/visa/agilent_pna.py

```diff
@@ -117,16 +129,28 @@ def read_raw_data(self, meas_name=''):
         """
         if meas_name:
             self.selected_measure = meas_name
+            
+        # get the endianess of the output so that the data is read properly
+        data_endianess = self._pna.data_endianess
+        if data_endianess == 'little':
+            is_big_endian = False
+        elif data_endianess == 'big':
+            is_big_endian = True
+        else:
+            raise InstrIOError(cleandoc('''Agilent PNA did not define 
+                                        endianess properly'''))
```

Comment:
You are already doing those checks in the property no need to duplicate them.
```suggestion
is_big_endian = self._pna.data_endianess == "big"
```

Reply:
Nice, thank you